pkgs/development/libraries/gstreamer/viperfx/default.nix

```diff
@@ -8,7 +8,7 @@
 }:
 stdenv.mkDerivation rec {
   pname = "gst-plugins-viperfx";
-  version = "unstable-2020-9-20";
+  version = "1.24.9";
```

Comment:
There is no such tag in the upstream
https://github.com/Audio4Linux/gst-plugin-viperfx

Reply:
I should have noticed that didn't update the hash. I expected `nix-update` to throw in error in such a case, but it must not.